### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo-Encoder#438]
@DanielnetoDotCom 

I can't get past this "Processing..." bar, it doesn't show any error. How can I know what's going on?
![image](https://user-images.githubusercontent.com/37880073/171989686-cb9a4864-043e-490d-b0a3-a264f6ce1ba6.png)

**Comment by DanielnetoDotCom** [comment]
Please check Apache error log

**Comment by gbrn** [comment]
```
[Sat Jun 04 15:21:56.045878 2022] [mpm_prefork:notice] [pid 27741] AH00163: Apache/2.4.29 (Ubuntu) configured -- resuming normal operations
[Sat Jun 04 15:21:56.045952 2022] [core:notice] [pid 27741] AH00094: Command line: '/usr/sbin/apache2'
[Sat Jun 04 15:22:00.528196 2022] [php7:warn] [pid 27746] [client 177.149.157.189:52656] PHP Warning:  require_once(/var/www/html/AVideo-Encoder/view/../videos/configuration.php): failed to open stream: No such file or directory in /var/www/html/AVideo-Encoder/view/index.php on line 18
[Sat Jun 04 15:22:00.528253 2022] [php7:error] [pid 27746] [client 177.149.157.189:52656] PHP Fatal error:  require_once(): Failed opening required '/var/www/html/AVideo-Encoder/view/../videos/configuration.php' (include_path='.:/usr/share/php') in /var/www/html/AVideo-Encoder/view/index.php on line 18
[Sat Jun 04 15:22:00.712728 2022] [php7:warn] [pid 27746] [client 177.149.157.189:52656] PHP Warning:  mkdir(): Permission denied in /var/www/html/AVideo-Encoder/objects/functions.php on line 275
[Sat Jun 04 15:22:04.555679 2022] [php7:warn] [pid 27749] [client 207.241.232.90:56666] PHP Warning:  mkdir(): Permission denied in /var/www/html/AVideo-Encoder/objects/functions.php on line 275
[Sat Jun 04 16:01:01.076662 2022] [mpm_prefork:notice] [pid 27741] AH00169: caught SIGTERM, shutting down
[Sat Jun 04 16:01:01.215115 2022] [mpm_prefork:notice] [pid 27978] AH00163: Apache/2.4.29 (Ubuntu) configured -- resuming normal operations
[Sat Jun 04 16:01:01.215188 2022] [core:notice] [pid 27978] AH00094: Command line: '/usr/sbin/apache2'
```

**Comment by gbrn** [comment]
@DanielnetoDotCom I have already done and redone all the steps of the wiki and videos for installation. I have mounted and unmounted the server and the problem is always the same.
I'm using AWS. An attempt to get a solution from the public encoder, because I'm not able to use it due to current problems.

**Comment by DanielnetoDotCom** [comment]
seems to me you are missing some permisisons

try this 

`mkdir /var/www/html/AVideo-Encoder/videos`

`chmod -R 777 /var/www/html/AVideo-Encoder/videos`

make sure the database is not created on the MySQL and let the script installer create it

**Comment by gbrn** [comment]
Yes, I run the permissions on the initial install.

But about the MySQL database, there's a detail, I'm using a database from another server, outside of AWS. But I'm letting the script create everything, I just indicate the URL, database name, username, password.

Do you think this works?

Have you ever tested an installation on AWS?

**Comment by DanielnetoDotCom** [comment]
> But about the MySQL database, there's a detail, I'm using a database from another server, outside of AWS. But I'm letting the script create everything, I just indicate the URL, database name, username, password.

Try to modify the script to point to the remote database

> Have you ever tested an installation on AWS?

Yes, I did install encoders on AWS many times

**Comment by gbrn** [comment]
As for modifying the Script, you say after running the installer? Why can't I find the files created on installation. 
Apparently it doesn't quite create the files.

**Comment by gbrn** [comment]
@DanielnetoDotCom Doing a search in Issues, I saw that these errors are recurrent.

I'm using Ubuntu 18... I'll try with Ubuntu 20.

Could you tell me why these specific errors happen?

`PHP Warning:  require_once(/var/www/html/AVideo-Encoder/view/../videos/configuration.php): failed to open stream: No such file or directory in /var/www/html/AVideo-Encoder/view/index.php on line 18`

`PHP Fatal error:  require_once(): Failed opening required '/var/www/html/AVideo-Encoder/view/../videos/configuration.php' (include_path='.:/usr/share/php') in /var/www/html/AVideo-Encoder/view/index.php on line 18`

`PHP Warning:  mkdir(): Permission denied in /var/www/html/AVideo-Encoder/objects/functions.php on line 275`

**Comment by DanielnetoDotCom** [comment]
That is why I wrote you this

https://github.com/WWBN/AVideo-Encoder/issues/438#issuecomment-1147439172

**Comment by gbrn** [comment]
I verified that this command is not accepted:
![image](https://user-images.githubusercontent.com/37880073/172249106-848cc0c4-cf3c-4ccf-b15b-4885f7b9aee3.png)

`chown www-data:www-data /var/www/html/AVideo-Encoder/videos && chmod 755 /var/www/html/AVideo-Encoder/videos`

`chown: changing ownership of '/var/www/html/AVideo-Encoder/videos': Operation not permitted`

**Comment by gbrn** [comment]
Now using a new Ubuntu 20 server... same error...

I executed all the permissions given, configured Apache as requested:

```
/var/www/html/AVideo-Encoder$ ls -l
total 96
-rwxrwxrwx 1 root     root        23 Jun  6 20:44 CNAME
-rwxrwxrwx 1 root     root      2964 Jun  6 20:44 Dockerfile
-rwxrwxrwx 1 root     root     34520 Jun  6 20:44 LICENSE
-rwxrwxrwx 1 root     root      8580 Jun  6 20:44 README.md
-rwxrwxrwx 1 root     root        26 Jun  6 20:44 _config.yml
drwxrwxrwx 3 root     root      4096 Jun  6 20:44 deploy
-rwxrwxrwx 1 root     root      3797 Jun  6 20:44 index.php
drwxrwxrwx 2 root     root      4096 Jun  6 20:44 install
drwxrwxrwx 2 root     root      4096 Jun  6 20:44 model
drwxrwxrwx 2 root     root      4096 Jun  6 20:44 nbproject
drwxrwxrwx 2 root     root      4096 Jun  6 20:44 objects
drwxrwxrwx 2 root     root      4096 Jun  6 20:44 update
drwxrwxrwx 2 www-data www-data  4096 Jun  6 20:51 videos
drwxrwxrwx 8 root     root      4096 Jun  6 20:44 view
```

Can you help me to verify if these permissions are enough for the system?



`PHP Warning:  require_once(/var/www/html/AVideo-Encoder/view/../videos/configuration.php): failed to open stream: No such file or directory in /var/www/html/AVideo-Encoder/view/index.php on line 18`

`PHP Fatal error:  require_once(): Failed opening required '/var/www/html/AVideo-Encoder/view/../videos/configuration.php' (include_path='.:/usr/share/php') in /var/www/html/AVideo-Encoder/view/index.php on line 18`

`PHP Warning:  mkdir(): Permission denied in /var/www/html/AVideo-Encoder/objects/functions.php on line 275`

**Comment by DanielnetoDotCom** [comment]
the permission seems to be correct. so I am not sure what is failing

check the videos/cofiguration.php file permission.

**Comment by LightRO** [comment]
I have the same Issue but what i have seen is following Message in the console from Web Browser:

`Access to XMLHttpRequest at 'https://tv.xxx.de/login' from origin 'https://encoder.xxx.de' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header contains multiple values 'https://encoder.xxx.de, *', but only one is allowed.`
